gauntlet/packages/gauntlet-solana-contracts/src/commands/contracts/store/setLoweringAccessController.ts

```diff
@@ -0,0 +1,62 @@
+import { Result } from '@chainlink/gauntlet-core'
+import { logger, prompt } from '@chainlink/gauntlet-core/dist/utils'
+import { SolanaCommand, TransactionResponse } from '@chainlink/gauntlet-solana'
+import { PublicKey } from '@solana/web3.js'
+import { CONTRACT_LIST, getContract } from '../../../lib/contracts'
+
+export default class SetAccessController extends SolanaCommand {
```

Comment:
Shouldn't this be renamed?

```suggestion
export default class SetLoweringAccessController extends SolanaCommand {
```